### DISCUSSION TITLE
間隔號採用 katakana middle dot

### DISCUSSION THREAD

**Discussion by hfhchan** [sparanoid/chinese-copywriting-guidelines#196] [Ideas]
Unicode 的 middle dot 與 hyphenation point 都是半形，但 CNS11643 與 HKSCS 分別將後者與前者訂為全形，但字體一於少理。

建議一律改用 U+30FB katakana middle dot。在任何支援該字的字體，均顯視為全形。

**Comment by sparanoid** [comment]
感覺一律使用 katakana middle dot 這種「用日文標點強制全形」的做法有點「矯枉過正」了，可能需要一個更好的解決方案

**Comment by hfhchan** [comment]
> In the end of 2015, Ministry of Education Taiwan hold several meetings to
> publish new version of "Handbook of Punctuations". The new version will add
> Unicode code points to punctuations. We have a proposal to use U+30FB for
> separation mark, encourage font foundries to include this glyph in Traditional
> Chinese font, and input method developers to map to it.

請參見 http://www.unicode.org/review/pri316/

**Comment by xfq** [comment]
供参考：在W3C的[中文排版需求](https://w3c.github.io/clreq/#id86)里，这个码位目前是不推荐使用的。

**Comment by lumynou5** [comment]
從語義（不是中文標點）來看有點怪，但為了排版維持全形可能可以接受。
